jupyter/pytorch/ubi9-python-3.9/Pipfile

```diff
@@ -12,44 +12,49 @@ name = "pytorch"
 
```

Comment:
We need to update the source link from: `url = "https://download.pytorch.org/whl/cu121"` to  `url = "https://download.pytorch.org/whl/cu121"`